### PR TITLE
Update Certify project code for Soc RSE membership

### PR DESCRIPTION
**Pull request by dsj976**
Certify makes you to enter a project code when filling in the expenses form.

## Summary
I have updated the project code necessary to claim back the annual Soc RSE membership fee.

### CODE REVIEW

File: content/docs/onboarding/society_of_research_software_engineering.md [hunk 1]
```diff
@@ -41,4 +41,4 @@ SocRSE membership is considered a professional subscription,
 which you're entitled to claim back as a REG member.
 
 You should claim it back quickly after payment.
-Use Certify to claim it back and don't select a project when making your claim.
+Use Certify to claim it back, and use the `R-RSE-001` project code when prompted.
```
Comment (JimMadge) [inline content/docs/onboarding/society_of_research_software_engineering.md:44]:
```suggestion
Use Certify to claim it back and use the `R-RSE-001` project code when prompted.
```

I don't think commas are commonly used to separate independent clauses in (British) English.

**Review by JimMadge — CHANGES_REQUESTED** [review]
Just a grammatical suggestion.

Should check with someone who knows whether this is the correct process.

**Comment by dsj976** [comment]
@martintoreilly could you confirm that this is the correct process to claim the Soc RSE annual membership fee on Certify?

**Comment by martintoreilly** [comment]
> @martintoreilly could you confirm that this is the correct process to claim the Soc RSE annual membership fee on Certify?

Yes it is (see the REG open wiki page on [reclaiming expenses](https://github.com/alan-turing-institute/research-engineering-group/wiki/Reclaiming-out-of-pocket-expenses)). I've added the full text in the Certify project name / code drop down for ease of reference.

**Comment by martintoreilly** [comment]
Do we want to cross-reference the REG open wiki (or just move the detail into the Handbook)? I'm not sure where the line is for things that are very Turing specific and not likely to be adopted / reused elsewhere and / or increasing risk of social engineering / exploitation of compromised credentials by openly documenting internal processes.

**Comment by dsj976** [comment]
> Do we want to cross-reference the REG open wiki (or just move the detail into the Handbook)? I'm not sure where the line is for things that are very Turing specific and not likely to be adopted / reused elsewhere and / or increasing risk of social engineering / exploitation of compromised credentials by openly documenting internal processes.

I personally think we should cross-reference the REG wiki here, no need to make project codes publicly available

**Comment by DavidBeavan** [comment]
> Just a grammatical suggestion.

I'm not getting prescriptive here, but does the handbook have a style guide for language? Like a publication would? [The Guardian one](https://www.theguardian.com/guardian-observer-style-guide-a) is quite accessible.

**Comment by JimMadge** [comment]
> > Just a grammatical suggestion.
> 
> I'm not getting prescriptive here, but does the handbook have a style guide for language? Like a publication would? [The Guardian one](https://www.theguardian.com/guardian-observer-style-guide-a) is quite accessible.

Not for language, spelling or grammar. There is some guidance on styling Markdown.

Feel free to propose one in a PR, it would be good to have an authority on what is right to remove the ambiguity. I don't think we want to write our own though.

The Guardian style doesn't seem so easy to navigate for grammatical questions. There is a lot about how to style the names of brands, people, organisations which isn't so useful here.

**Comment by dsj976** [comment]
I guess this PR is now ready to be merged? I will merge it by the end of the day, unless anyone has any reservations.

**Review by JimMadge — APPROVED** [review]

**Pull request merged**